src/ServiceInsight.Tests/AutoApprover.cs

```diff
@@ -0,0 +1,41 @@
+﻿namespace ServiceInsight.Tests
+{
+    using System.IO;
+    using ApprovalTests.Core;
+
+    class AutoApprover : IReporterWithApprovalPower
```

Comment:
Is there any value in keeping this class? We're not using it anywhere, since the corresponding attribute is commented out in `XamlIconsTests`.